### PR TITLE
Feature/prom and zipkin trace integration

### PR DESCRIPTION
**Pull request by domtronn**
- This adds a new wrapper function into `koa-zipkin` to easily create trace spans
- It upgrades the `koa-core` library to backwards compatabily generate these traces
- It updates the example server to provide this functionality
- It adds a docker compose file with `zipkin` to allow for encapsulated testing

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+10
```
Comment (phelma) [inline .nvmrc:1]: not 14?

Comment (domtronn) [reply .nvmrc:1]: No, annoyingly - it's 10.

Comment (domtronn) [reply .nvmrc:1]: Never been updated and don't want to deal with that upgrade in this PR - better to at least state what it was built against

**Review by phelma — APPROVED** [review]

File: __config__/prometheus-config.production.json [hunk 2]
```diff
@@ -64,6 +64,13 @@
     "help": "Number of errors per second per request",
     "type": "Meter"
   },
+  {
+    "name": "trace_duration_seconds",
+    "help": "The time in seconds for traces to complete",
+    "type": "Histogram",
+    "labelNames": ["span"],
+    "buckets": [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0]
+  },
```
Comment (joshcroad) [inline __config__/prometheus-config.production.json:73]: Is it worth adding this to the `koa-prometheus` library directly?
https://github.com/uswitch/koa-core/blob/master/packages/koa-prometheus/src/koa-prometheus.standards.json

Comment (domtronn) [reply __config__/prometheus-config.production.json:73]: Yes, I think you're right - are most people using the standards JSON?

Comment (joshcroad) [reply __config__/prometheus-config.production.json:73]: Good point, the default is `false` for loading standards - maybe worth adding it to the default one then

**Pull request merged**